src/main/java/com/example/draftspringboot/controller/DraftController.java

```diff
@@ -29,9 +25,12 @@ public DraftResponse getById(@PathVariable @Parameter(description = "Id of somet
         return draftService.getById(id);
     }
 
-    @GetMapping
-    public List<DraftResponse> getAll() {
-        return draftService.getAll();
+    @Operation(summary = "Get list of something with pagination and search by part of name")
+    @GetMapping()
+    public Page<DraftResponse> searchByPartName(
+            @Parameter(description = "index of page") @RequestParam(required = false) Integer index,
```

Comment:
I recomend to name this parameter as `page` or `offset` - depends on what particularly it contain
Look at
https://www.taniarascia.com/rest-api-sorting-filtering-pagination/
https://www.moesif.com/blog/technical/api-design/REST-API-Design-Filtering-Sorting-and-Pagination/

Reply:
Ok